### PR TITLE
Release 1.8.0 [MTT-9227]

### PR DESCRIPTION
**Pull request by RikuTheFuffs**
### Description
PR for v1.8.0 release.

### Issue Number(s)
- [MTT-9227](https://jira.unity3d.com/browse/MTT-9227)

### Contribution checklist
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.8.0] 2024-10-21
```
Comment (fernando-cortez) [inline CHANGELOG.md:3]: Just a comment to update the date.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+### Multiplayer Use Cases
+
+#### Added
+- The last page of the "Data/Event Synchronization" tutorial redirects to the "Proximity Checks" tutorial
+- Imported TextMeshPro essentials so you don't have to do it
+
+#### Changed
+- Updated project to Unity 6 (and packages to the latest stable versions)
```
Comment (fernando-cortez) [inline CHANGELOG.md:12]: It'd be helpful to highlight the relevant project dependencies that were updated with from & to version.

**Review by fernando-cortez — DISMISSED** [review]
Pretty much good to go!

File: CHANGELOG.md (hunk 2 above)
Comment (RikuTheFuffs) [reply CHANGELOG.md:12]: Do we still need that? (I added a comment about this in the 1.8.0 release doc, we can have the conversation there).
In the meantime, to unblock the release, the full list here

**Pull request merged**